src/stylesheets/components/_disclaimer.scss

```diff
@@ -8,6 +8,16 @@
   }
 }
 
+.usa-disclaimer-inner {
```

Comment:
This is changed to have a centered grid rather than the disclaimer links floated to left and right.